go/cmd/sqlflow/main.go

```diff
@@ -460,11 +463,18 @@ func processOptions(opts *options) {
 		err = listModels(opts)
 	case opts.List && opts.Repo:
 		err = listRepos(opts)
+	case opts.Get && opts.Model:
+		filename, err = downloadModelFromDB(opts)
 	default:
 		err = runSQLFlowClient(opts)
 	}
 	if err != nil {
 		log.Printf("Failed due to %v", err)
+	} else {
+		// output downloaded model path if no error.
+		if opts.Get && opts.Model {
+			fmt.Printf("model \"%s\" downloaded successfully at %s\n", opts.ModelName, filename)
```

Comment:
How about put this piece of code in `downloadModelFromDB`, so the `processOptions` only do dispatch and don't do anything else.

Reply:
Good point.